city_scrapers/spiders/il_elections.py

```diff
@@ -0,0 +1,148 @@
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlElectionsSpider(CityScrapersSpider):
+    name = "il_elections"
+    agency = "Illinois State Board of Elections"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.elections.il.gov/AboutTheBoard/MeetingMinutesAll.aspx"]
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def __init__(self):
+        self.meeting_minutes = dict()
+        self.addresses = dict()
+        super().__init__()
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        # parse meeting minutes and addresses first to associate them with the agenda
+        self._parse_minutes(response)
+        self._parse_addresses(response)
```

Comment:
I don't think this needs to be parsed in the minutes response, since the same footer is on the agenda page and moving it there could simplify this a bit